src/services/review/ReviewRequestService.ts

```diff
@@ -224,9 +224,13 @@ export default class ReviewRequestService {
     const { siteName, isomerUserId: userId } = sessionData
 
     // Find all review requests associated with the site
-    const requests = await this.repository.findAll({
+    // TODO: Note this needs to be findAll when we reach a stage of allowing
+    // multiple open PRs simultaneously
+    // Current behaviour returns the newest Open PR based on created_at field
+    const request = await this.repository.findOne({
       where: {
         siteId: site.id,
+        reviewStatus: ReviewRequestStatus.Open,
```

Comment:
Sanity check here: do we only want `open` reviews? Dont we also need `Approved` and/or `closed`ones because they are open but have not been merged in yet?

Reply:
@kishore03109 During discussion we agreed to fetch the latest open PR, which I felt referred to the PRs in "open" status. 

@seaerchin  Do we need to displays Approved/Closed PRs which are not yet merged?

Reply:
good catch @kishore03109! i searched the FE for this - we'd need to give both `APPROVED | OPEN` states. could i also trouble you (@harishv7) to rename this + the route to `listValidReviewRequests` for clarity?

Reply:
done with renaming and fetching for both approved + open states. As discussed,

There can only exist 1 active PR which is in either Open or Approved state. The new logic will fetch the most recently created PR DB entry which is in Open/Approved state

Reply:
wait sanity check, the code here still filters by `reviewStatus: ReviewRequestStatus.Open,` and not by  approved + open state, is this intended?

Reply:
@kishore03109 Looking at https://github.com/isomerpages/isomercms-backend/pull/769/files#diff-ec790b65dee9ec111ee3c30c98efd651b561a57b31c998d06b200db95e68255f, code is updated to

```
const request = await this.repository.findOne({
      where: {
        siteId: site.id,
        [Op.or]: [
          {
            reviewStatus: ReviewRequestStatus.Open,
          },
          { reviewStatus: ReviewRequestStatus.Approved },
        ],
      },
```